eventually/tests/unittests/authentication/testforgetpassword.py

```diff
@@ -0,0 +1,114 @@
+"""
+Forget Password test module
+===========================
+"""
+
+import json
+
+from django.test import TestCase, Client
+
+from authentication.views import CustomUser
+from utils.jwttoken import create_token
+
+VALID_EMAIL_TOKEN = create_token({"user_id":100, 'email': 'john@email.com'}, 60 * 60)
+INVALID_EMAIL_TOKEN = create_token({"user_id":999, 'email': 'john@email.com'}, 60 * 60)
+INVALID_TOKEN = "eyJ1c2VyX2lkIjoxMDAsImVtYWlsIjoiam9obkBlbWFpbC5"
+
+class TestForgetPassword(TestCase):
+    """ Class for Forget Password tests """
+
+    def setUp(self):
+        """ Set up test browser client """
+        self.client = Client()
+
+        user = CustomUser.objects.create(id=100, email="john@email.com")
+        user.set_password = "QwErTy12234"
+        user.save()
+
+
+    def test_post_positive(self):
+        """ Positive test for forget password POST function """
+
+        link = '/api/v1/user/forget_password/'
```

Comment:
maybe it is better to use `url` name in that case, in order to have all test files in one style.

Reply:
Makes sense. Changed names.